packages/infrastructure/kube_airbyte/main.tf

```diff
@@ -1375,6 +1694,11 @@ module "image_cache" {
       registry   = "docker.io"
       repository = "temporalio/auto-setup"
       tag        = "1.26"
+    },
+    {
```

Comment:
I don't really feed like it is necessary to have any of these in the cache. The cache isn't infinite and it doesn't seem like decreasing the startup time of these pods by a minute will have any tangible impact on the functionality or UX of the system since airbyte is mostly a set-and-forget type of system.